next/components/ui/MapSection/MapSection.tsx

```diff
@@ -143,7 +143,7 @@ const MapSection = ({ branches, mapboxAccessToken, title, altDesign = false }: M
             return (
               <CardWrapper
                 key={branch.id}
-                className={cx('relative', {
+                className={cx('relative ring-inset', {
```

Comment:
Not ideal, but it works :)
![image](https://github.com/user-attachments/assets/3525b4e6-e9ff-45a6-839b-b8d19fc8162e)